### PR TITLE
Responsive Form

### PR DESCRIPTION
**Pull request by ghost**
As a mobile user, I would like the form labels (e.g. "Title", "Description") to be responsive to different screen sizes so that the form label is always readable.

Acceptance criteria:
Form labels are readable for all screen sizes

### CODE REVIEW

File: app/views/articles/_form.html.erb [hunk 1]
```diff
@@ -6,14 +6,14 @@
       <%= form_with(model: @article, class: "shadow p-3 mb-3 form-bg rounded", local: true) do |f| %>  
         <div class="form-group row">
             <%= f.label :title, class: "col-2 col-form-label text-light font-weight-bold" %>
-            <div class="col-10">
+            <div class="col-md-10 col-sm-12">
```
Comment (aboutsimon) [inline app/views/articles/_form.html.erb]: 100% width is the default, you just need to specify

```suggestion
            <div class="col-md-10">
```

File: app/views/articles/_form.html.erb [hunk 2]
```diff
@@ -6,14 +6,14 @@
       <%= form_with(model: @article, class: "shadow p-3 mb-3 form-bg rounded", local: true) do |f| %>  
         <div class="form-group row">
             <%= f.label :title, class: "col-2 col-form-label text-light font-weight-bold" %>
```
Comment (aboutsimon) [inline app/views/articles/_form.html.erb]: the corresponding change is missing here

```suggestion
            <%= f.label :title, class: "col-md-2 col-form-label text-light font-weight-bold" %>
```

File: app/views/users/_form.html.erb [hunk 3]
```diff
@@ -1,34 +1,34 @@
 <div class="container">
   <div class="row justify-content-center">
-    <div class= "col-10">
+    <div class= "col-md-10 col-sm-12">
       <%= render 'shared/errors', obj: @user %>
 
       <%= form_with(model: @user, class: "shadow p-3 mb-3 form-bg rounded", local: true) do |f| %>  
         <div class="form-group row">
             <%= f.label :username, class: "col-2 col-form-label text-light font-weight-bold" %>
```
Comment (aboutsimon) [inline app/views/users/_form.html.erb]: same as above

File: app/views/users/_form.html.erb [hunk 4]
```diff
@@ -1,34 +1,34 @@
...
-            <div class="col-10">
+            <div class="col-md-10 col-sm-12">
```
Comment (aboutsimon) [inline app/views/users/_form.html.erb]: same

File: app/views/sessions/new.html.erb [hunk 5]
```diff
@@ -3,21 +3,21 @@
 
 <div class="container">
   <div class="row justify-content-center">
-    <div class= "col-10">
+    <div class= "col-md-10 col-sm-12">
```
Comment (aboutsimon) [inline app/views/sessions/new.html.erb]: Not really related to this PR, but i feel like using `col` classes to decrease the container width on all pages (as far as i can tell) is not the right tool for the job. Not completely wrong, either, but if you want to improve this, you could check out the bootstrap doc, on [how to theme bootstrap](https://getbootstrap.com/docs/4.0/getting-started/theming/) and [specifically the container width](https://getbootstrap.com/docs/4.0/layout/grid/#grid-tiers).

**Review by aboutsimon — COMMENTED** [review]
left some thoughts, hope you find them useful!

File: app/views/articles/_form.html.erb (hunk 1 above)
Comment (ghost) [reply app/views/articles/_form.html.erb]: @aboutsimon aha so you mean I do not have to write `col-sm-12` only `col-md-10` ?

File: app/views/sessions/new.html.erb (hunk 5 above)
Comment (ghost) [reply app/views/sessions/new.html.erb]: Ah I see what you mean, this is a good point. If, however, `col` is good enough, then I would prefer to leave it like this for now :smiley_cat:

File: app/views/articles/_form.html.erb (hunk 1 above)
Comment (aboutsimon) [reply app/views/articles/_form.html.erb]: exactly

**Review by aboutsimon — APPROVED** [review]
go for it!

**Pull request merged**